react/src/components/SessionDetailContent.tsx

```diff
@@ -0,0 +1,196 @@
+                  id
+                  image {
+                    id
+                    name
+                    architecture
+                    tag
+                  }
+                }
+              }
+            }
+            vfolder_mounts
+            created_at @required(action: NONE)
+            terminated_at
+            scaling_group
+            agent_ids
+            requested_slots
+
+            ...SessionStatusTagFragment
+            ...SessionActionButtonsFragment
+            ...SessionTypeTagFragment
+            ...EditableSessionNameFragment
+            ...SessionReservationFragment
+          }
+          legacy_session: compute_session(id: $uuid) {
+            image
+            mounts
+            user_email
+            architecture
+          }
+        }
+      `,
+      {
+        id: toGlobalId('ComputeSessionNode', id),
+        uuid: id,
+        project_id: currentProject.id,
+      },
+      {
+        fetchPolicy: 'network-only',
+        fetchKey: fetchKey,
+      },
+    );
+
+  const imageFullName =
+    legacy_session?.image &&
+    legacy_session?.architecture &&
+    legacy_session.image + '@' + legacy_session.architecture;
+  return session ? (
+    <Flex direction="column" gap={'sm'} align="stretch">
+      {/* {JSON.stringify(compute_session_node.requested_slots, null, 2)} */}
+      <Flex
+        direction="row"
+        justify="between"
+        align="start"
+        style={{
+          alignSelf: 'stretch',
+        }}
+      >
+        <EditableSessionName
+          sessionFrgmt={session}
+          component={Title}
+          level={3}
+          style={{
+            margin: 0,
+            flex: 1,
+          }}
+          editable={false}
+        />
+        <Button.Group size="large">
+          <SessionActionButtons sessionFrgmt={session} />
+        </Button.Group>
+      </Flex>
+
+      <Descriptions bordered column={md ? 2 : 1}>
+        <Descriptions.Item label={t('session.SessionId')} span={md ? 2 : 1}>
+          <Typography.Text copyable style={{ fontFamily: 'monospace' }}>
+            {session.row_id}
+          </Typography.Text>
+        </Descriptions.Item>
+        {(userRole === 'admin' || userRole === 'superadmin') && (
+          <Descriptions.Item label={t('credential.UserID')} span={md ? 2 : 1}>
+            {legacy_session?.user_email}
+          </Descriptions.Item>
+        )}
+        <Descriptions.Item
+          label={t('session.Status')}
+          contentStyle={{ display: 'flex', gap: token.marginSM }}
+        >
+          <SessionStatusTag sessionFrgmt={session} />
+          {/* <Button type="text" icon={<TriangleAlertIcon />} /> */}
+        </Descriptions.Item>
+        <Descriptions.Item label={t('session.SessionType')}>
+          <SessionTypeTag sessionFrgmt={session} />
+        </Descriptions.Item>
+        <Descriptions.Item label={t('session.launcher.Environments')}>
+          {imageFullName ? (
+            <Flex gap={'sm'}>
```

Comment:
```suggestion
            <Flex gap={'sm'} wrap={'wrap'}>
```
Let's add `wrap` for a narrow screen.
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/2HueYSdFvL8pOB5mgrUQ/2c9293cb-a91c-4f20-8bf3-c37a5b864877.png)

Reply:
Thank you for the suggestion. Even after applying your suggestion, there is still something to improve regarding the layout, and this part should be changed after #2800. I think it's better to handle this in a separate PR after merging #2800.